winit-win32/src/lib.rs

```diff
@@ -1,25 +1,43 @@
-//! # Windows
+//! # Winit Win32 / Windows backend
 //!
 //! The supported OS version is Windows 7 or higher, though Windows 10 is
 //! tested regularly.
+#![cfg(target_os = "windows")] // FIXME(madsmtm): Allow compiling on all platforms.
+
+#[macro_use]
+mod util;
+mod dark_mode;
+mod definitions;
+mod dpi;
+mod drop_handler;
+mod event_loop;
+mod icon;
+mod ime;
+mod keyboard;
+mod keyboard_layout;
+mod monitor;
+pub mod raw_input;
```

Comment:
should this be `pub mod`?

Reply:
Probably should, felt easier to just not think about it yet.

Reply:
I went with exposing the things we want to expose individually, instead of exposing the entire module.

Reply:
I'd generally pub use only things that implements the trait, but also very carefully. In general, I'd probably later do a pass on what should be public and what not to avoid diff growing like we have now.

Reply:
Oh, sorry, I misread your intent here - you did _not_ want it to be `pub mod`